include/liblwm2m.h

```diff
@@ -157,23 +157,29 @@ bool lwm2m_session_is_equal(void * session1, void * session2, void * userData);
 #define COAP_IGNORE                     (uint8_t)0x01
 #define COAP_RETRANSMISSION             (uint8_t)0x02
 
-#define COAP_201_CREATED                (uint8_t)0x41
-#define COAP_202_DELETED                (uint8_t)0x42
-#define COAP_204_CHANGED                (uint8_t)0x44
-#define COAP_205_CONTENT                (uint8_t)0x45
-#define COAP_231_CONTINUE               (uint8_t)0x5F
-#define COAP_400_BAD_REQUEST            (uint8_t)0x80
-#define COAP_401_UNAUTHORIZED           (uint8_t)0x81
-#define COAP_402_BAD_OPTION             (uint8_t)0x82
-#define COAP_404_NOT_FOUND              (uint8_t)0x84
-#define COAP_405_METHOD_NOT_ALLOWED     (uint8_t)0x85
-#define COAP_406_NOT_ACCEPTABLE         (uint8_t)0x86
-#define COAP_408_REQ_ENTITY_INCOMPLETE  (uint8_t)0x88
-#define COAP_412_PRECONDITION_FAILED    (uint8_t)0x8C
-#define COAP_413_ENTITY_TOO_LARGE       (uint8_t)0x8D
-#define COAP_500_INTERNAL_SERVER_ERROR  (uint8_t)0xA0
-#define COAP_501_NOT_IMPLEMENTED        (uint8_t)0xA1
-#define COAP_503_SERVICE_UNAVAILABLE    (uint8_t)0xA3
+#define COAP_201_CREATED                    (uint8_t)0x41
+#define COAP_202_DELETED                    (uint8_t)0x42
+#define COAP_203_VALID                      (uint8_t)0x43
+#define COAP_204_CHANGED                    (uint8_t)0x44
+#define COAP_205_CONTENT                    (uint8_t)0x45
+#define COAP_231_CONTINUE                   (uint8_t)0x5F
+#define COAP_400_BAD_REQUEST                (uint8_t)0x80
+#define COAP_401_UNAUTHORIZED               (uint8_t)0x81
+#define COAP_402_BAD_OPTION                 (uint8_t)0x82
+#define COAP_403_FORBIDDEN                  (uint8_t)0x83
+#define COAP_404_NOT_FOUND                  (uint8_t)0x84
+#define COAP_405_METHOD_NOT_ALLOWED         (uint8_t)0x85
+#define COAP_406_NOT_ACCEPTABLE             (uint8_t)0x86
+#define COAP_408_REQ_ENTITY_INCOMPLETE      (uint8_t)0x88
+#define COAP_412_PRECONDITION_FAILED        (uint8_t)0x8C
+#define COAP_413_ENTITY_TOO_LARGE           (uint8_t)0x8D
+#define COAP_415_UNSUPPORTED_CONTENT_FORMAT (uint8_t)0x8F
+#define COAP_500_INTERNAL_SERVER_ERROR      (uint8_t)0xA0
+#define COAP_501_NOT_IMPLEMENTED            (uint8_t)0xA1
+#define COAP_502_BAD_GATEWAY                (uint8_t)0xA2
+#define COAP_503_SERVICE_UNAVAILABLE        (uint8_t)0xA3
+#define COAP_504_GATEWAY_TIMEOUT            (uint8_t)0xA4
+#define COAP_505_PROXYING_NOT_SUPPORTED     (uint8_t)0xA5
```

Comment:
Commit description says that those codes are from [Block-Wise Transfers in the Constrained Application Protocol (CoAP)](https://www.rfc-editor.org/rfc/rfc7959.html).

However, for example 5.05 Proxying Not Supported I found in [RFC 7252](https://datatracker.ietf.org/doc/html/rfc7252#section-5.9.3.6) instead. Any chance the reference git mixed up?

Reply:
Anyway I updated the commit description

Reply:
The wording "updated by" suggests that it (mainly) adds to, not replaces an RFC.

https://www.rfc-editor.org/rfc-index2.html:

> "Updated by xxxx" refers to RFCs that have updated but not replaced this one.

So, thanks for updating.